R/system.R

```diff
@@ -274,11 +274,11 @@ getDBConnection <- function(type, host, port, databaseName, username, password){
   if(!requireNamespace("DBI")){stop("package DBI must be installed.")}
   drv = NULL
   conn = NULL
-  if(type == "mysql" || type == "aurora"){
+  if(type == "mysql" | type == "aurora"){
     drv <- DBI::dbDriver("MySQL")
     conn = RMySQL::dbConnect(drv, dbname = databaseName, username = username,
                              password = password, host = host, port = port)
-  } else if (type == "postgres" || type == "redshift"){
+  } else if (type == "postgres" | type == "redshift" | type == "hp_vertica"){
```

Comment:
Okay, I was always using "|" because it also work on one element vector but using "||" seems more apropriate if we are sure that it's one length vector. I'll use || here.
http://stackoverflow.com/questions/6558921/r-boolean-operators-and

I'll change the name too